pom.xml

```diff
@@ -124,7 +124,7 @@
     </dependencies>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <openapi-generator-version>5.0.1</openapi-generator-version>
+        <openapi-generator-version>5.1.1</openapi-generator-version>
```

Comment:
Also need to bump the version of the CLI in the Makefile which is what's used to actually build the clients.